src/translations/hi/translation.json

```diff
@@ -105,5 +105,46 @@
   "Dinner 🍽️": "रात का खाना 🍽️",
   "Walk 🚶🏽": "टहलना 🚶🏽",
   "Water the plants 🪴 indoors": "पौधों 🪴 को घर के अंदर पानी दें",
-  "Me time 🧘🏽😌": "मेरे लिए समय 🧘🏽😌"
+  "Me time 🧘🏽😌": "मेरे लिए समय 🧘🏽😌",
+  "dontAskAgain": "क्या आप इस कार्रवाई के लिए फिर से पूछना नहीं चाहेंगे?",
+  "note": "नोट",
+  "completeGoal": "लक्ष्य पूरा करें",
+  "deleteGoal": "लक्ष्य हटाएं",
+  "shareGoal": "लक्ष्य साझा करें",
+  "chooseContact": "संपर्क चुनें",
+  "collaborateOnGoal": "लक्ष्य पर सहयोग करें",
+  "confirm": "पुष्टि करें",
+  "cancel": "रद्द करें",
+  "goal": {
+    "archive": {
+      "header": "क्या आप इस लक्ष्य को पूरा करना चाहते हैं?",
+      "note": "यह लक्ष्य सबलक्ष्यों के साथ संग्रहण में जाएगा। आप इसे संग्रहण से कभी भी पुनर्स्थापित कर सकते हैं।"
+    },
+    "delete": {
+      "header": "क्या आप इस लक्ष्य को हटाना चाहते हैं?",
+      "note": "आप इसे फिर से प्राप्त नहीं कर पाएंगे।"
+    },
+    "shareAnonymously": {
+      "header": "क्या आप इस लक्ष्य को नाम छिपाकर साझा करना चाहते हैं?",
+      "note": "केवल यह लक्ष्य और इसके समय संबंधित गुणों को सार्वजनिक रूप से साझा किया जाएगा :)"
+    },
+    "shareWithOne": {
+      "header": "क्या आप इस लक्ष्य को अपने संपर्कों में से किसी एक के साथ साझा करना चाहते हैं?",
+      "note": "आपके द्वारा इस लक्ष्य पर की गई हर परिवर्तन उस व्यक्ति के साथ साझा किया जाएगा।"
+    }
+  },
+  "collaboration": {
+    "colabRequest": {
+      "header": "क्या आप सहयोग करना चाहते हैं?",
+      "note": "सहयोग के दौरान, यह लक्ष्य आपके मेरे लक्ष्यों में जोड़ा जाएगा और आपके द्वारा किए गए किसी भी परिवर्तन को दूसरे व्यक्ति तक संचारित किया जाएगा।"
+    },
+    "delete": {
+      "header": "क्या आप इस लक्ष्य को हटाना चाहते हैं?",
+      "note": "इससे सहयोग समाप्त हो जाएगा और आपको कोई अद्यतन प्राप्त नहीं होगा।"
+    },
+    "archive": {
+      "header": "क्या आप इस लक्ष्य को संग्रहित करना चाहते हैं?",
+      "note": "इससे सहयोग समाप्त हो जाएगा और आपको कोई अद्यतन प्राप्त नहीं होगा।"
+    }
+  }
 }
```

Comment:
Please add EOL in all reqd files.

Reply:
I am not able to understand what EOL means. Can you give me a example?

Reply:
Please configure your eslint with the project so that these issues can be fixed in one go.

Reply:
nvm i've resolved for now. can't keep PR(s) open for so long :)